app/models/chargeback_rate_detail.rb

```diff
@@ -16,11 +16,18 @@ def find_rate(value)
     fixed_rate = 0.0
     variable_rate = 0.0
     chargeback_tiers.each do |tier|
-      next if value < rate_adjustment(tier.start)
-      next if value >= rate_adjustment(tier.finish)
-      fixed_rate = tier.fixed_rate
-      variable_rate = tier.variable_rate
-      break
+      if tier.starts_with_zero?
```

Comment:
@tledesma for which cases is this condition ? (which do not to reflect condition from else leg)

and what about to use select method and rewrite condition to `if the value is included` ? something like, maybe it is more readable:

`chargeback_tiers.select{|tier| value > rate_adjustment(tier.start) && value <= rate_adjustment(tier.finish) }`

what do you think ?